### ISSUE TITLE
Coffee not brewing ☕️

### ISSUE THREAD

**Issue by petersolimine** [Coframe/coffee#53]
First of all—very cool project!

I am trying to set it up and run it within a nextjs app.

I ran my nextjs app, then opened a new terminal, cd into the nextjs directory, and ran the docker container with my OpenAI API key.

However, when I add a <Coffee> component and save my file, nothing changes. There appear to be no changes in my output, either. 

I followed the instructions exactly — what might be the cause?

<img width="344" alt="Screen Shot 2023-12-15 at 3 21 44 PM" src="https://github.com/Coframe/coffee/assets/16181699/ddb26214-4ce1-4e09-b35d-92fce5c2f876">
<img width="570" alt="Screen Shot 2023-12-15 at 3 21 57 PM" src="https://github.com/Coframe/coffee/assets/16181699/141e1a8d-e478-48b3-abd0-a4f90ebc7606">

**Comment by 1um** [comment]
Nice, do you run Coffee docker from same folder as your FE code?
```
docker run --pull=always -it -e OPENAI_API_KEY=${OPENAI_API_KEY} -v $(pwd):/mount coframe/coffee:latest
```
In this command `-v $(pwd):/mount` is mirroring all files from current folder `$(pwd)` to `/mount` folder in docker.
I'll add check that `/mount` is not empty on stratup.

**Comment by 1um** [comment]
Scratch that, something else is an issue. What is filename of file you edit and OS you are using?

**Comment by petersolimine** [comment]
Yes, same directory. I'm on MacOS Monterey, M1 Chip. Filename is `src/app/components/destinations.tsx`

**Comment by saikrishnaklu1993** [comment]
![image](https://github.com/Coframe/coffee/assets/153780407/c9489061-ea56-4ca3-a879-1c0488429d89)
![Screenshot 2023-12-16 170650](https://github.com/Coframe/coffee/assets/153780407/c9bb8d7f-1084-4991-9363-c227302c63a3)

Have same issue on windows - it doesn't recognize the coffee component. 

on windows(inside the app dir):
docker run --pull=always -it -e OPENAI_API_KEY="xxxx" -v "$(pwd):/mount" coframe/coffee:latest

@petersolimine  great thanks for opensourcing!! appreciate your help in setup.

**Comment by sam-shridhar1950f** [comment]
Same issue here on Windows. The coffee component isn't recognized despite running docker in the same directory.

**Comment by OmmAshutosh** [comment]
ssue could be related to permissions or the location of the Compose binary.Here are a few steps you can take to troubleshoot this issue:
1) which docker-compose; this should return the path to the Docker Compose binary. Make sure that this path is included in your system's PATH environment variable.
2) ls -l $(which docker-compose)
3) sudo chmod +x $(which docker-compose)
4) docker-compose --version
5) sudo chmod +x $(which docker-compose)
6) docker-compose --version
Check for any conflicting Docker installations: Make sure that there are no other Docker installations on your system that could be conflicting with Docker Desktop. If there are, you may need to uninstall them before reinstalling Docker Desktop.
